src/pages/terms-and-conditions/_clients-grid.js

```diff
@@ -60,7 +60,7 @@ const IconGrid = () => {
             content: localize('What you’re agreeing to when you sign up to use Deriv'),
             non_eu_links: [
                 {
-                    url: '/tnc/general-terms.pdf',
+                    url: '/tnc/general-terms-v1.0.pdf',
```

Comment:
Hi @fiona-deriv, I think we can just rename the new file. No change in this file is needed.

Reply:
Okay, I'll revert that